src/dinero.js

```diff
@@ -876,6 +876,21 @@ const Dinero = options => {
         currency,
         precision
       }
+    },
+    /**
+     * Return the object's data as an object literal.
+     *
+     * Alias of {@link module:Dinero~toObject toObject}.
+     * It is defined so that calling `JSON.stringify` on a Dinero object will automatically extract the relevant data.
+     *
+     * @example
+     * // returns '{"amount":500,"currency":"EUR","precision":2}'
+     * JSON.stringify(Dinero({ amount: 500, currency: 'EUR', precision: 2 }))
+     *
+     * @return {Object}
+     */
+    toJSON() {
```

Comment:
This needs a reference in l. 35, in the list of **Conversion & formatting** methods.